doc/main.html

```diff
@@ -527,7 +527,8 @@ <h4>Normative references section</h4>
           <li>Each <code>li</code> element may contain one <code>a</code> element that contains a location where the reference can
           be retrieved.</li>
         </ul>
-
+          <p class="note">It is recommended to use undated references (to designate using the latest version), except for when needed as per <a href="#bib-iso-directives-part2"></a>. For SMPTE documents, see <a href="#bib-ag-29"></a> for the specification of persistent link URIs for both "latest" and specific versions of a document referenced.</p>
```

Comment:
Why a note if it is recommended? I would either make it a _should_ statement or use _to be_.

I also think these two separate concepts: (a) use undated references and (b) what URL to use for SMPTE documents (which can be a note).

Reply:
Noted and updated in https://github.com/SMPTE/html-pub/pull/314/commits/1c915cc2e922f5fd5f105d680a8b4075fbcd996a